debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/SnapshotReaderIT.java

```diff
@@ -31,12 +36,16 @@
  * @author Randall Hauch
  *
  */
+@RunWith(Arquillian.class)
 public class SnapshotReaderIT {
 
     private static final Path DB_HISTORY_PATH = Testing.Files.createTestingPath("file-db-history-snapshot.txt").toAbsolutePath();
     private static final String DB_NAME = "connector_test_ro";
     private static final String LOGICAL_NAME = "logical_server_name";
 
+    @ArquillianResource
```

Comment:
Is there a way to obtain this resource from Arquillian without having to inject it into all the tests? I.e. can it somehow be injected directly into `MySQLCube` itself?